packages/emails/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dx": "docker compose up -d"
+    "dx": "docker compose up -d || docker-compose up -d"
```

Comment:
Do we actually use docker in the calcom/emails package? If not, should we remove it instead?